js/core/config.js

```diff
@@ -21,6 +21,7 @@ const config = {
     forceIsoDateParsing: true,
     wrapActionsBeforeExecute: true,
     useLegacyStoreResult: false,
+    useRenovatedComponents: false,
```

Comment:
@RomanTsukanov check this new private field name in the global config pls.

Reply:
This one is fine.